axis_linear_x: 1
axis_linear_y: 0
axis_angular: 3

speed_up: 2        # Speed increased by 20%
speed_down: 0      # Speed decreased by 20%
deadman: 4
operating_mode: 6  # differential drive mode and mecanum drive mode (Not used now)

max_linear_x: 1.5
max_linear_y: 0
max_angular: 3.0